packages/fxa-settings/src/components/Settings/PageRecoveryPhoneRemove/index.tsx

```diff
@@ -26,13 +27,37 @@ const sumoTwoStepLink = (
 const PageRecoveryPhoneRemove = (props: RouteComponentProps) => {
   const navigate = useNavigate();
   const account = useAccount();
+  const alertBar = useAlertBar();
+  const ftlMsgResolver = useFtlMsgResolver();
+
   // TODO, actually get this number back and format it
-  const formattedFullPhoneNumber = '+1 (555) 555-5555';
+  const formattedFullPhoneNumber = '+1 ••• ••••';
```

Comment:
I think auth client supports fetching this value? Is there a reason not to do this here?

Reply:
I was hoping to rebase on Lauren PR, but I can do as follow up in https://github.com/mozilla/fxa/pull/18287 too.

Reply:
I think we want the full number here instead of just last four digits (based on Figma), so that might be a separate request? I'm still on the fence about this. Kinda feel like we could display the whole number all the time in Settings when the session is 2FA verified.

Reply:
@vbudhram I can fix this in my PR since I have some similar fixes. I'm going to have merge conflicts with your GQL recovery phone update that's in the demo branch, so I'll rebase on yours.

Reply:
Just a heads up I'm updating this in my branch, but I'm just going to just display the unformatted phone number. I'm going to file a follow up issue from my PR around what we get back when the session isn't verified + that `national_format` bit so it should get taken care of there.